undefined symbol: mysql_real_escape_string_quote

I am getting an error:

```
ImportError: /opt/conda/envs/_test/lib/python3.5/site-packages/_mysql.cpython-35m-x86_64-linux-gnu.so: undefined symbol: mysql_real_escape_string_quote
```

When trying to build a conda package with mysqlclient master branch. I think this is caused by commit #109?

```
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Dversion_info=(1,3,7,'final',1) -D__version__=1.3.7 -I/opt/conda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho
/include -I/opt/conda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/include/python3.5m -c _mysql.c -o build/temp.linux-x86_64-3.5/_mysql.o
In file included from /opt/conda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/include/mysql.h:64,
                 from _mysql.c:30:
/opt/conda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/include/mysql/client_plugin.h:97: warning: function declaration isn’t a prototype
In file included from /opt/conda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/include/mysql.h:64,
                 from _mysql.c:30:
/opt/conda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/include/mysql/client_plugin.h:107: warning: function declaration isn’t a prototype
_mysql.c: In function ‘_mysql_escape_string’:
_mysql.c:1068: warning: implicit declaration of function ‘mysql_real_escape_string_quote
```

It seems right header is used, but `mysql_real_escape_string_quote` is used. Strange...
What is your MYSQL_VERSION_ID in `/opt/conda/envs/.../include/mysql_version.h`?

--- a/_mysql.c
+++ b/_mysql.c
@@ -1064,7 +1064,7 @@ _mysql_escape_string(
 	if (self && PyModule_Check((PyObject*)self))
 		self = NULL;
 	if (self && self->open) {
-#if MYSQL_VERSION_ID >= 50706
+#if MYSQL_VERSION_ID >= 50707
 		len = mysql_real_escape_string_quote(&(self->connection), out, in, size, '\'');
 #else
 		len = mysql_real_escape_string(&(self->connection), out, in, size);
@@ -1122,7 +1122,7 @@ _mysql_string_literal(
 	out = PyBytes_AS_STRING(str);
 	check_server_init(NULL);
 	if (self && self->open) {
-#if MYSQL_VERSION_ID >= 50706
+#if MYSQL_VERSION_ID >= 50707
 		len = mysql_real_escape_string_quote(&(self->connection), out+1, in, size, '\'');
 #else
 		len = mysql_real_escape_string(&(self->connection), out+1, in, size);

